src/time.cpp

```diff
@@ -34,6 +35,7 @@ namespace g3 {
          //                    ... also ... This is way more buffer space then we need
 
          auto success = std::strftime(buffer, size, c_time_format, tmb);
+         // TODO: incorrect code
```

Comment:
// TODO: incorrect code
         if (0 == success)
         {
            // assert is always false because of if condition
            assert((0 != success) && "strftime fails with illegal formatting");